go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/helmfile/chartify
 go 1.20
 
 require (
+	github.com/Masterminds/semver v1.5.0
```

Comment:
Why not reuse the v3 version that's already pulled in?

Reply:
@reegnz oh. good catch. thanks. I will update it.